backend/src/main/java/WebInterfaceImpl.java

```diff
@@ -1,9 +1,27 @@
+import model.DigitalPart;
+import storage.DatabaseConnector;
+
+import java.util.List;
+
 /**
  * Created by danie on 2017-09-28.
  */
 public class WebInterfaceImpl implements WebInterface {
+    DatabaseConnector dbConnector;
+
     @Override
-    public String getAllPrintsForStl(String stlID) {
-        return "Fuck yes! You gave me the ID " + stlID + ".";
+    public String dummy(String id) {
+        return "Fuck yes! You gave me the ID " + id + ".";
+    }
+
+    @Override
+    public DigitalPart getDigitalPart(int digitalPartID) {
+        //TODO implement this properly
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)